sdk/storage/test-resources.json

```diff
@@ -276,6 +272,14 @@
     "AZURE_CLIENT_SECRET": {
       "type": "string",
       "value": "[parameters('testApplicationSecret')]"
+    },
+    "DFS_ACCOUNT_NAME": {
```

Comment:
We've learned from running live tests in sovereign clouds (e.g. Azure China Cloud and US Government Cloud) that environment variables with storage account names (but no URLs) tend to mean that tests have the URL suffixes hardcoded and the test code is generating the storage URL itself. This means that tests fail in sovereign clouds because they're accessing the wrong URL. 

This problem is further compounded by the fact that ARM templates for storage accounts don't appear to have any easy methods for retrieving the full URL of the storage account. 

This test-resources.json is already hard coded to use the Azure Cloud (see the connection string generated on line 262 with 'core.windows.net'). However, if you have a chance, we prefer that the ARM template generate the URL or provide enough pieces to the test code so that the test code generates the correct URL. 

```
"parameters": { 
  ... 
  "storageEndpointSuffix": { 
    "type": "string",
    "defaultValue": "core.windows.net"
  }
  ...
},
...
"outputs": { 
  ...
  "DFS_URL": { 
    "type": "string",
    "value": "[format('https://{0}.dfs.{1}/', variables('dataLakeAccountName'), parameters('storageEndpointSuffix')]"
  },
  ...
} 
```

Using this pattern it's possible to specify different storage endpoint suffixes and have this work with sovereign clouds. We may have to parameterize the `dfs` part of the URL in the future but this pattern simplifies that process and keeps us from having to open up bugs for SDK engineers to comb through the test code and update how URLs are generated.

Reply:
Thanks for bringing this up. Where are we gonna use this `DFS_URL`? Will our pipeline run in sovereign clouds?

Reply:
We are already trying to support running tests in sovereign clouds using the existing pipelines in a branch that is configured for that purpose. 

The `DFS_URL` would be used in place of `DFS_ACCOUNT_NAME` in the test code instead of building a URL that assumes the endpoint is `{DFS_ACCOUNT_NAME}.dfs.core.windows.net`

In this PR I only see `DFS_ACCOUNT_NAME` mentioned in a post-deployment script but I assume that tests would also use that information to connect to a live resource.

Reply:
I see. This requires changing our test utilities. Probably we should log another issue.